TasksList.dart

```diff
@@ -0,0 +1,67 @@
+import 'TaskClass.dart';
+import 'dart:io';
+
+void main(){
+ try {
+  List<Map<int,TaskInfo>>  AddedTasks = [];
+  while(true){
+    
+   
+    print("Welcome Welcome:)\n~~~~~~~~~~~\nHere we can help you  manange your tasks. ");
+    print("What would you like to do?\n1.Add a task to your list\n2.Veiw a task from your list\n3.Delete a task from your list\n4.Exist\n");
+    String ? entry = stdin.readLineSync();
+    TaskInfo AddedTask = TaskInfo('','','',false);
+    if(entry == "1"){
+      print("WELL DO\n Please enetr your task info:\n");
+      
+      stdout.write("Task Number: ");
+      int taskNum = int.tryParse(stdin.readLineSync() ?? '') ?? 0;
+      AddedTask.Num = taskNum.toString();
+
+      stdout.write("Task Name: ");
+      AddedTask.Name = stdin.readLineSync() ?? '';
+      stdout.write("Task description: ");
+      AddedTask.Description = stdin.readLineSync() ?? '';
+      stdout.write("Task status -false or true-: ");
+      AddedTask.CompletedOrNot = (stdin.readLineSync()?.toLowerCase() == 'true');
```

Comment:
checking for empty is also important
```suggestion
      stdout.write("Task Name: ");
      String name = stdin.readLineSync() ?? '';
      if(name.trim().isEmpty)
      {
        print("Name cannot be empty");
      }
      AddedTask.Name = stdin.readLineSync() ?? '';

      stdout.write("Task description: ");
      AddedTask.Description = stdin.readLineSync() ?? '';
      stdout.write("Task status -false or true-: ");
      AddedTask.CompletedOrNot = (stdin.readLineSync()?.toLowerCase() == 'true');
```